contracts/Ovm_SpokePool.sol

```diff
@@ -8,6 +8,8 @@ import "@eth-optimism/contracts/libraries/bridge/CrossDomainEnabled.sol";
 import "@eth-optimism/contracts/libraries/constants/Lib_PredeployAddresses.sol";
 import "@eth-optimism/contracts/L2/messaging/IL2ERC20Bridge.sol";
 
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
Don't need to import this as its already imported in `SpokePool.sol`

Reply:
We actually don't have to import it anywhere SafeErc20 is imported as that also pulls it in. I deleted it in SpokePOol and Ovm_SpokePool as well